FlightJournal.Web/Views/TrainingLog/_PartialTrainingLogView.cshtml

```diff
@@ -7,12 +7,10 @@
         <thead>
         <tr>
             <th>@_("Date")</th>
-            @*<th>@_("Plane")</th>*@
-            @*<th>@_("Front seat")</th>*@
             <th>@_("Back seat")</th>
```

Comment:
Front seat ? instead of Back seat ?

Reply:
No, that's intentional.
This is used for 'recent flights' for a student pilot, so we already know who's in the front seat.
Saves space in the UI.